src/components/StaticTableNew/StaticTableNew.tsx

```diff
@@ -0,0 +1,242 @@
+import React, { useState, useEffect, useMemo } from 'react';
```

Comment:
I haven't looked deeply at this, I wonder what we will want for the best abstraction moving forward.